device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/td3-a7050cx3-32s-32x100G.config.bcm

```diff
@@ -1,3 +1,13 @@
+host_as_route_disable=1
```

Comment:
For all TD3 config.bcm files will also require a line similar to the one here:

sai_load_hw_config=/etc/bcm56870_a0/b870.6.6.1/

I will disclose the specific path once we have validate this change.